satisfactory/egg-satisfactory.json

```diff
@@ -4,7 +4,7 @@
         "version": "PTDL_v2",
         "update_url": null
     },
-    "exported_at": "2024-06-01T00:04:44+00:00",
+    "exported_at": "2024-06-08T04:41:17+00:00",
     "name": "Satisfactory",
     "author": "rehlmgaming@gmail.com",
```

Comment:
While this is being updated, could you please update this to be red_thirten@yahoo.com please? And the Pterodactyl Egg too. The current email is now defunct. (I need to update this on my other Eggs as well soon)

Reply:
fixed in both eggs